BlackNectar/BlackNectar/DistanceCalculation Class.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  DistanceCalculation Class.swift
+//  BlackNectar
+//
+//  Created by K on 12/29/16.
+//  Copyright © 2016 Black Whole. All rights reserved.
+//
+
+import Foundation
+import CoreLocation
+
+class DistanceCalculation {
+    
+    func getDistance(userLocation: CLLocationCoordinate2D, storeLocation: CLLocationCoordinate2D) -> Double {
+        
+        let userCoordinateToLocation = CLLocation(latitude: userLocation.latitude, longitude: userLocation.longitude)
```

Comment:
Name these accordingly: `first`, `second`, or `from`, `to`, or `origin` & `destination`.